app/views/miq_template/show.html.haml

```diff
@@ -11,7 +11,7 @@
     - case @showtype
     - when "details"
       = render :partial => "layouts/gtl", :locals => {:action_url => @lastaction}
-    - when "miq_proxies", "storage_extents", "storage_systems", "ontap_storage_volumes", "ontap_file_shares"
+    - when "miq_proxies", "storage_extents"
```

Comment:
why no `"storage_systems"`? - seems like ems_cluster_controller still sets that (and you're not removing it from the other places)